src/nn/prepo.rs

```diff
@@ -14,99 +19,109 @@ extern "C" {
         this: *mut PlayReport,
         arg1: *const u8,
     ) -> root::Result;
-}
-extern "C" {
+
     #[link_name = "\u{1}_ZN2nn5prepo10PlayReport9SetBufferEv"]
     pub fn PlayReport_SetBuffer(this: *mut PlayReport)
         -> root::Result;
-}
-extern "C" {
+
     #[link_name = "\u{1}_ZN2nn5prepo10PlayReport3AddEPKcl"]
     pub fn PlayReport_Add(
```

Comment:
Instead of Add/Add1/Add2, rename according to what they expect, such as AddUsize, AddFloat, AddDouble, AddString, ...